.nvmrc

```diff
@@ -1 +1 @@
-16.13
+16.13.2
```

Comment:
Not strictly related to this change, but [`asdf`](https://asdf-vm.com/) requires an exact version.